include/matx/transforms/solve/solve_cudss.h

```diff
@@ -219,54 +220,81 @@ using gemm_cudss_cache_t =
     std::unordered_map<SolveCUDSSParams_t, std::any, SolveCUDSSParamsKeyHash,
                        SolveCUDSSParamsKeyEq>;
 
+template <typename Op>
+__MATX_INLINE__ auto getSolveSupportedTensor(const Op &in,
+                                             cudaStream_t stream) {
+  const auto func = [&]() {
+    if constexpr (is_tensor_view_v<Op>)
+      return in.Stride(Op::Rank() - 1) == 1;
+    return true;
```

Comment:
I think this might generate a warning with some compilers if you don't wrap in an `else`

Reply:
even with the fall into return?

Reply:
Yes, because it's not mutually exclusive, as written, without the `else`. It's really a compiler problem, but you could get a dead code line on the bottom return if the constexpr `if` is `true`

Reply:
Ah, I see. That makes sense (although then still here, the Stride is never fully known at compile-time, right? Still, I can change all my instances.

Reply:
the stride doesn't matter here. it's that the if constexpr needs that else following it to make the two paths mutually exclusive